generic/MAPL_Generic.F90

```diff
@@ -758,8 +758,8 @@ end subroutine MAPL_GenericSetServices
    ! !IROUTINE: MAPL_GenericInitialize -- Initializes the component and its children
 
    ! !INTERFACE:
-   recursive subroutine MAPL_GenericInitialize ( GC, IMPORT, EXPORT, CLOCK, RC )
-
+   recursive subroutine MAPL_GenericInitialize ( GC, import, EXPORT, CLOCK, RC )
```

Comment:
Should all the arguments be lowercase if you change one?

Reply:
Not - sure how that happened - was not on my agenda today.   Probably best to just leave the PR as changed.   consistent capitalization is already out between my style and that of others.  Someday ...

Reply:
okay. Resolving.